WooCommerce/Classes/ViewRelated/InAppFeedback/InAppFeedbackCardViewController.swift

```diff
@@ -42,6 +42,10 @@ private extension InAppFeedbackCardViewController {
     func configureDidNotLikeButton() {
         didNotLikeButton.applySecondaryButtonStyle()
         didNotLikeButton.setTitle(Localization.couldBeBetter, for: .normal)
+        didNotLikeButton.on(.touchUpInside) { _ in
+            let surveyNavigation = SurveyCoordinatingController.init(survey: .inAppFeedback)
+            self.present(surveyNavigation, animated: true, completion: nil)
```

Comment:
I always have doubts about these kinds of functions, I tend to think that if a button is tapped it's because self exists 😅 
I'll change it to be on the safe side